doc/releasenotes/15_0_0_summary.md

```diff
@@ -1,10 +1,24 @@
+## Summary
+
+- [Vindex Interface](#vindex-interface)
+- [LogStats Table and Keyspace deprecated](#logstats-table-and-keyspace-deprecated)
+- [Command-line syntax deprecations](#command-line-syntax-deprecations)
+- [New command line flags and behavior](#new-command-line-flags-and-behavior)
+- [Online DDL changes](#online-ddl-changes)
+- [Tablet throttler](#tablet-throttler)
+- [VDiff2](#vdiff2)
+- [Mysql Compatibility](#mysql-compatibility)
+- [Durability Policy](#durability-policy)
```

Comment:
How do you feel about doing different levels of bullet points, like so:

```markdown
- #### [Breaking Changes](#breaking-change)
  - [Vindex Interface](#vindex-interface)
```
------ 
Preview:
- #### Breaking Changes
  - Vindex Interface

Reply:
I'm open to this idea :) We can do it one time towards the end of the release.